tmail-backend/pom.xml

```diff
@@ -236,647 +195,49 @@
-                <version>${james.version}</version>
-            </dependency>
-            <dependency>
-                <groupId>${james.groupId}</groupId>
-                <artifactId>james-server-cassandra-rabbitmq-guice</artifactId>
-                <version>${james.version}</version>
-                <type>test-jar</type>
-            </dependency>
-            <dependency>
-                <groupId>${james.groupId}</groupId>
-                <artifactId>james-server-cli</artifactId>
-                <version>${james.version}</version>
-            </dependency>
-            <dependency>
-                <groupId>${james.groupId}</groupId>
-                <artifactId>james-server-data-api</artifactId>
-                <version>${james.version}</version>
-            </dependency>
-            <dependency>
-                <groupId>${james.groupId}</groupId>
-                <artifactId>james-server-data-ldap</artifactId>
-                <version>${james.version}</version>
-                <type>test-jar</type>
-            </dependency>
-            <dependency>
-                <groupId>${james.groupId}</groupId>
-                <artifactId>james-server-data-memory</artifactId>
-                <version>${james.version}</version>
-            </dependency>
-            <dependency>
-                <groupId>${james.groupId}</groupId>
-                <artifactId>james-server-guice-cassandra</artifactId>
-                <version>${james.version}</version>
-            </dependency>
-            <dependency>
-                <groupId>${james.groupId}</groupId>
-                <artifactId>james-server-guice-configuration</artifactId>
-                <version>${james.version}</version>
-            </dependency>
-            <dependency>
-                <groupId>${james.groupId}</groupId>
-                <artifactId>james-server-guice-common</artifactId>
+                <artifactId>james-project</artifactId>
                 <version>${james.version}</version>
+                <type>pom</type>
+                <scope>import</scope>
             </dependency>
             <dependency>
                 <groupId>${james.groupId}</groupId>
-                <artifactId>james-server-guice-common</artifactId>
-                <type>test-jar</type>
+                <artifactId>james-server-guice</artifactId>
                 <version>${james.version}</version>
+                <type>pom</type>
+                <scope>import</scope>
             </dependency>
             <dependency>
                 <groupId>${james.groupId}</groupId>
-                <artifactId>james-server-guice-data-ldap</artifactId>
-                <version>${james.version}</version>
-            </dependency>
-            <dependency>
-                <groupId>org.apache.james</groupId>
-                <artifactId>james-server-guice-distributed</artifactId>
+                <artifactId>jmap-rfc-8621-integration-tests-common</artifactId>
```

Comment:
Do we still need this then?

Reply:
Not defined in the BOM POM thus we need an explicit statement.